src/components/facet/facetVertical.js

```diff
@@ -453,6 +453,7 @@ FacetVertical.prototype._updateSparkline = function() {
 			selectedSparklinePath.appendTo(sparkline);
 
 			totalSparklinePath[0].classList.add('facet-sparkline-total');
+            totalSparklinePath[0].classList.add('facetkey-sparkline-total');
```

Comment:
This merely adds a unique class that FacetKey can override using a precise bit of CSS. If this class is undefined in other projects it should do nothing.